packages/verification/src/Verification.vue

```diff
@@ -188,7 +188,7 @@ export default {
                 }
                 if (offchainVerification) {
                   // If it's already verified on blockchain, do not override all values; just issuerName & address can be taken from off-chain information
-                  if (!this.verificationItems[i].issuerName && offchainVerification.issuerName && offchainVerification.status === 'registered') {
+                  if (!this.verificationItems[i].issuerName && offchainVerification.issuerName && ['registering', 'revoking', 'revoked'].indexOf(offchainVerification.status) !== -1) {
```

Comment:
registered should also be included, right?

Reply:
Thanks for noticing: fb58af12ef0a40edea30a0ff92274b040237f59d